### PR TITLE
Add kingmaker Appendix Armies and show usable tactics in actions lists

### PR DESCRIPTION
**Pull request by TMunsie**
There isn't a ton of automation other than grants of actions as feats don't seem to construct army actions.

### CODE REVIEW

File: packs/kingmaker-bestiary/armies/lizardfolk-defenders.json [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+                    "value": []
+                },
+                "publication": {
+                    "license": "OGL",
+                    "remaster": false,
+                    "title": "Pathfinder Kingmaker"
+                },
+                "rules": [
+                    {
+                        "key": "FlatModifier",
```
Comment (simonward) [inline packs/kingmaker-bestiary/armies/lizardfolk-defenders.json:307]: Is this correct? Doesn't seem to match the description.

File: packs/kingmaker-features/army-tactics/swamp-charge.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            "value": []
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Kingmaker"
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
```
Comment (simonward) [inline packs/kingmaker-features/army-tactics/swamp-charge.json:32]: And here.

File: packs/kingmaker-features/army-tactics/swamp-dwellers.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            "value": []
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Kingmaker"
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
```
Comment (simonward) [inline packs/kingmaker-features/army-tactics/swamp-dwellers.json:32]: And here.

**Comment by CarlosFdez** [comment]
We should avoid adding NPC tactics and actions to the compendiums, since it'll confuse players trying to build their own armies.

**Comment by BernhardPosselt** [comment]
What's left to get this PR in? Any updates?

**Comment by TMunsie** [comment]
I'm leaving it up to Supe and Cora to edit and merge as Kingmaker was a commercial project and I'm not going to take away their license on how they want it presented.

**Pull request merged**